CHANGELOG.md

```diff
@@ -58,7 +58,6 @@ the latest cosmos proto versions from `v0.40.0-rc5` (sometimes called 'stargate-
 - Update to cosmos-sdk IBC proto version `v0.40.0-rc5` ([#451])
 
 - [relayer]
-  - Implement packet relaying ([#379]) 
```

Comment:
Not sure we should have this modification.

Reply:
hmm...I haven't changed that manually maybe got changed in a merge for some reason. I had to merge master a few times. Can manually revert if needed.

Reply:
Yes, I think we should:

- revert this change
- update the changelog to mention a fix for https://github.com/informalsystems/ibc-rs/issues/32

Reply:
Fixed with 37a48b10f15ca6be47b31a7ac239738ead2750ef